docs/how-to/craft/code/example-classic-confinement-autotools.yaml

```diff
@@ -0,0 +1,27 @@
+# autotools-classic-example, an example of a classically-confined project with Autotools
+
+name: autotools-classic-example
+version: "1.0"
+summary: An example snap for an autotools project
+description: |
+  Shows how to create a snap for an application that uses a Makefile for
+  build and installation.
+
+grade: devel
+base: core22
+confinement: classic
+
+apps:
+  # Using the name of the snap as the key name allows the program to be invoked by its
+  # name.
```

Comment:
pedantic: code boxes overflow long lines and a single word here looks awkward in the rendered doc.  I'd suggest splitting the comment in half across the two lines or make it a one-line comment. 

<img width="684" alt="image" src="https://github.com/user-attachments/assets/3a9cb07e-27ec-4de3-a1b4-843d872c5442" />

Reply:
Good catch: [d49dfb3](https://github.com/canonical/snapcraft/pull/5345/commits/d49dfb3e5f8e44dca56906c51ed9a8f43e9f83ac)

It's too bad that the theme styles that element for ~80.5 chars. And uses italics for comments (grumble).

FWIW, this code block boundary is arbitrary and favours large-screen devices. On a Galaxy S20 with default scaling, the full page width looks like this:

![image](https://github.com/user-attachments/assets/42f5dd91-804f-415e-b61e-7fa91fe985ab)

It's a repeat of the terminal width problem and has no general solution.

Reply:
Bummer, at least the overflow more noticeable on a mobile/small screen.  

Your change looks good.